ui/src/widgets/ui-text-input/UITextInput.vue

```diff
@@ -110,7 +109,36 @@ export default {
             }
         }
     },
+    created () {
+        // can't do this in setup as we are using custom onInput function that needs access to 'this'
+        useDataTracker(this.id, this.onInput, this.onLoad, null)
+
+        // let Node-RED know that this widget has loaded
+        this.$socket.emit('widget-load', this.id)
```

Comment:
this is handled as part of the data tracker, so shouldn't be required?

Reply:
Yes, you are probably right, just copied the function from another widget (don't recall which one) that had that. But indeed the `data tracker` on the `onMounted` always call this.

Will fix it.